CHANGES.txt

```diff
@@ -24,6 +24,9 @@ Changes
 Fixes
 =====
 
+ - Fixed a bug which would incorrectly rewrite a join when table stats
```

Comment:
I guess it threw an error so I would rephrase it to: **Fixed a bug that caused an error to be thrown in case of a ``JOIN`` query when table stats indicated that the left table of a join is smaller than the right due to a wrong rewrite.**

Reply:
No, it produced incorrect results.

Reply:
Then: **Fixed a bug that caused incorrect results to be returned in case of a ``JOIN`` query when table stats indicated that the left table of a join is smaller than the right due to a wrong plan rewrite.**

Reply:
I think I'd skip the technical part of the join rewrite: **Fixed a bug that caused incorrect results to be returned for JOIN queries when the table stats indicated that the left table of a join is smaller than the right.**